spec/features/user_can_sign_up_for_an_account_spec.rb

```diff
@@ -0,0 +1,34 @@
+feature 'User can sign up for an account' do
+  context 'on sign-up form' do
+    before do
+      visit root_path
+      click_on 'Sign up'
+    end
+
+    it 'displays the sign-up form' do
+      expect(page).to have_content 'Sign up'
+      expect(page).to have_field 'Name'
+      expect(page).to have_field 'Email'
+      expect(page).to have_field 'Password'
+      expect(page).to have_field 'Password confirmation'
+    end
+
+    it 'display Create button' do
+      expect(page).to have_button 'Create'
+    end
+  end
```

Comment:
This test is not really necessary, by testing that you can fill in all of those fields you are also testing that they are there 👍🏻

Reply:
We took away that test block.